Packs/CortexXDR/Classifiers/classifier-PaloAltoNetworks_CortexXDR.json

```diff
@@ -1,220 +1,14 @@
 {
-	"brandName": "Cortex XDR - IR",
-	"custom": true,
+	"fromVersion": "6.0.0",
+	"name": "Cortex XDR - Classifier",
+    "type": "classification",
+    "id": "Cortex XDR - IR",
```

Comment:
indentation

Reply:
i dont know why git shows it like that, but it looks fine on my ide
![image](https://user-images.githubusercontent.com/31018228/83383159-37fb1580-a3ed-11ea-8d81-1e20d05cddf3.png)

Reply:
I think because of the difference between tab and 4 spaces.

Reply:
you're right, fixed